src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt

```diff
@@ -54,13 +52,7 @@ private fun getVisitingName(): String {
 
 var beenAfkFor = SimpleTimeMark.now()
 
-fun getPetDisplay(): String = PetAPI.currentPet?.let {
-    val colorCode = it.substring(1..2).first()
-    val petName = it.substring(2).removeColor()
-    val petLevel = getCurrentPet()?.petLevel?.currentLevel ?: "?"
-
-    "[Lvl $petLevel] ${colorCodeToRarity(colorCode)} $petName"
-} ?: "No pet equipped"
+fun getPetDisplay(): String = PetAPI.pet?.rawPetName ?: "No pet equipped"
```

Comment:
this function is public visible. put that into a discord sepcific utils or inline